src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1106,6 +1128,44 @@ static bool isOrContainsErrorType(TypeSymbol typeSymbol)
             }
         }
 
+        private static bool CheckValidNullableOverride(
```

Comment:
>CheckValidNullableOverride [](start = 28, length = 26)

nit: name of the method is a bit misleading as it doesn't actually perform a check. Maybe `ShouldCheck...`?